Userland/Libraries/LibGfx/TIFFGenerator.py

```diff
@@ -124,6 +126,9 @@ class ExtraSample(EnumWithExportName):
     Tag('285', [TIFFType.ASCII], [], None, "PageName"),
     Tag('296', [TIFFType.UnsignedShort], [1], ResolutionUnit.Inch, "ResolutionUnit", ResolutionUnit),
     Tag('339', [TIFFType.UnsignedShort], [], SampleFormat.Unsigned, "SampleFormat", SampleFormat),
```

Comment:
Not a part of this PR, but this one is out of order.

Reply:
:redlinus: